[2023-10-02T13:00:57.750+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world.build_stage_1 manual__2023-10-02T12:53:48.446756+00:00 [queued]>
[2023-10-02T13:00:57.758+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world.build_stage_1 manual__2023-10-02T12:53:48.446756+00:00 [queued]>
[2023-10-02T13:00:57.758+0000] {taskinstance.py:1331} INFO - Starting attempt 2 of 2
[2023-10-02T13:00:57.777+0000] {taskinstance.py:1350} INFO - Executing <Task(BashOperator): build_stage_1> on 2023-10-02 12:53:48.446756+00:00
[2023-10-02T13:00:57.782+0000] {standard_task_runner.py:57} INFO - Started process 3841 to run task
[2023-10-02T13:00:57.784+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'hello_world', 'build_stage_1', 'manual__2023-10-02T12:53:48.446756+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/dag_1.py', '--cfg-path', '/tmp/tmpszvyccep']
[2023-10-02T13:00:57.784+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask build_stage_1
[2023-10-02T13:00:57.823+0000] {task_command.py:410} INFO - Running <TaskInstance: hello_world.build_stage_1 manual__2023-10-02T12:53:48.446756+00:00 [running]> on host 00a598f19374
[2023-10-02T13:00:57.905+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world' AIRFLOW_CTX_TASK_ID='build_stage_1' AIRFLOW_CTX_EXECUTION_DATE='2023-10-02T12:53:48.446756+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-02T12:53:48.446756+00:00'
[2023-10-02T13:00:57.906+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-10-02T13:00:57.906+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo <Task(BashOperator): build_stage_1> is running in the <DAG: hello_world> pipeline']
[2023-10-02T13:00:57.929+0000] {subprocess.py:86} INFO - Output:
[2023-10-02T13:00:57.979+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `('
[2023-10-02T13:00:57.979+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo <Task(BashOperator): build_stage_1> is running in the <DAG: hello_world> pipeline'
[2023-10-02T13:00:57.980+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-10-02T13:00:57.989+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-10-02T13:00:58.001+0000] {taskinstance.py:1373} INFO - Marking task as FAILED. dag_id=hello_world, task_id=build_stage_1, execution_date=20231002T125348, start_date=20231002T130057, end_date=20231002T130058
[2023-10-02T13:00:58.016+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 23 for task build_stage_1 (Bash command failed. The command returned a non-zero exit code 2.; 3841)
[2023-10-02T13:00:58.036+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-10-02T13:00:58.052+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
